rand_distr/src/normal_inverse_gaussian.rs

```diff
@@ -66,6 +66,25 @@ where
             inverse_gaussian,
         })
     }
+
+    /// Returns the alpha parameter (`alpha`) of the distribution.
+    pub fn alpha(&self) -> F {
+        // By definition:
+        //      gamma = sqrt(alpha^2 - beta^2) with alpha > 0
+        //      mu = 1 / gamma
+        // Therefore:
+        //      mu = 1 / sqrt(alpha^2 - beta^2)
+        //      mu^2 = 1 / (alpha^2 - beta^2)
+        //      1 / mu^2 = alpha^2 - beta^2
+        //      1 / mu^2 + beta^2 = alpha^2
+        //      sqrt(1 / mu^2 + beta^2) = alpha with alpha > 0
+        F::sqrt(self.inverse_gaussian.mean().powi(2).recip() + self.beta.powi(2))
```

Comment:
I think this computation may produce slightly different `alpha` than one used for distribution initialization. Maybe it's worth to add a comment in the docs about it? Same for a number of other getters.

Reply:
Indeed, I've commented in #1233 about the issue, I've proposed a set of alternatives, that's why this PR is just a draft.